nrf_security/doc/nrf_security.rst

```diff
@@ -216,17 +231,22 @@ Multiple backends
 ~~~~~~~~~~~~~~~~~
 AES core support can be configured by setting setting Kconfig variables according to the following table:
 
-+--------------+----------------+-------------------------------------------------+
-| Cipher mode  | Support        | Configurations                                  |
-+==============+================+=================================================+
-| ECB          | Glue           | CC310: :option:`CONFIG_CC310_MBEDTLS_AES_C`     |
-|              |                |                                                 |
-|              |                | Standard: :option:`CONFIG_VANILLA_MBEDTLS_AES_C`|
-+--------------+----------------+-------------------------------------------------+
++--------------+----------------+----------------------------------------------------+
+| Cipher mode  | Support        | Configurations                                     |
++==============+================+====================================================+
+| ECB          | Glue           | CC310: :option:`CONFIG_CC310_MBEDTLS_AES_C`        |
+|              |                |                                                    |
+|              |                | nrf_oberon: :option:`CONFIG_OBERON_MBEDTLS_AES_C`  |
```

Comment:
this option link doesn't resolve either

Reply:
This should hopefully been fixed now that we have the actual library in